### PR TITLE
Minor refinements to signal table

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/7957

## Testing
**URL to test:** 
https://7957-signal-refinements--atd-moped-main.netlify.app/moped/dashboard

**Steps to test:**

Change table to 30 rows
Reduce cell padding to 10px
Sort by last updated at
Set whitespace property for table headers to nowrap
Hide deleted projects from table
Change column names to reduce wrapping:
- "Targeted construction start" --> "Construction start"
- "Project completion date" --> "Project completion"
- "Last modified" --> "Modified"

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

**Comment by amenity** [comment]
👀  looking dandy! 

Since I only see 8 projects in the signals table on this Netlify deploy, I tried adding a few more: 
- [This one](https://7957-signal-refinements--atd-moped-main.netlify.app/moped/projects/367/) - started as non-signal project, then added a signal and set the project type to `Signal - New`
- [This one](https://7957-signal-refinements--atd-moped-main.netlify.app/moped/projects/368) - new project created from signal

Neither of these is showing on the table...why not? 🤔

File: moped-editor/src/queries/signals.js [hunk 1]
```diff
@@ -17,8 +17,10 @@ export const SIGNAL_PROJECTS_QUERY = gql`
             }
             status_id: { _neq: 4 }
           }
-        ]
+        ],
+        status_id: {_eq: 1}
```
Comment (mateoclarke) [inline moped-editor/src/queries/signals.js:21]: This line seems to be the rabbit hole that Amenity's question would take you down.

**Comment by mateoclarke** [comment]
Code looks good but going to wait to approve until the "Hide deleted projects from table" item that Amenity pointed out gets addressed. Seems like we might be missing a mutation on that `status_id` in the new project workflow

**Comment by chiaberry** [comment]
I was returning the wrong status_id. I forgot that deleted projects now have a status id of 6. Are there any other project statuses that shouldn't appear on the table?
Screen Shot 2021-12-15 at 10 05 44 AM

**Review by amenity — APPROVED** [review]
Perfect, thanks @chiaberry !

File: moped-editor/src/queries/signals.js [hunk 2]
```diff
@@ -17,8 +17,10 @@ export const SIGNAL_PROJECTS_QUERY = gql`
             }
             status_id: { _neq: 4 }
           }
-        ]
+        ],
+        status_id: {_neq: 6}
```
Comment (mateoclarke) [inline moped-editor/src/queries/signals.js:21]: 👍

**Pull request merged**